File: app/src/main/java/com/diyainfotech/covid19/MyApplication.java
package com.diyainfotech.covid19;

import android.app.Application;
import android.content.Context;

/**
 * @author sanjaypatel
 */
public class MyApplication extends Application {
    private static MyApplication application;

    @Override
    public void onCreate() {
        super.onCreate();
        application = this;
    }
    public static MyApplication getApplication() {
        return application;
    }
    public static Context getAppContext() {
        return application.getApplicationContext();
    }

}

File: app/src/main/java/com/diyainfotech/covid19/MyApplication.kt
package com.diyainfotech.covid19

import android.app.Application
import android.content.Context

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        application = this
    }

    companion object {
        var application: MyApplication? = null
            private set

        val appContext: Context
            get() = application!!.applicationContext
    }
}